mac/KeymanEngine4Mac/KeymanEngine4Mac/CoreWrapper/CoreWrapper.m

```diff
@@ -67,6 +64,38 @@ -(void)changeKeyboardWithKmxFilePath:(NSString*) path {
   }
 }
 
+-(CoreKeyboardInfo*) getKeyboardInfoForKmxFile:(NSString*)kmxFile {
+  NSArray *keyArray = [self getKeyArray];
+  CoreKeyboardInfo* info = [[CoreKeyboardInfo alloc] init:kmxFile keyArray: keyArray];
+  return info;
+}
+
+/**
+ * Get the list of Keys supported by the keyboard
+ */
+-(NSArray*)getKeyArray {
+  NSMutableArray *keyArray = [[NSMutableArray alloc] initWithCapacity:0];
+  km_core_keyboard_key *keyList;
+
+  km_core_status result = km_core_keyboard_get_key_list(self.coreKeyboard, &keyList);
```

Comment:
Is keyList leaked here?

Reply:
yes, adding a call to km_core_keyboard_key_list_dispose